### PR TITLE
Add Landespolizei des Fürstentums Liechtenstein

### PR DESCRIPTION
**Pull request by Double-A-92**

### CODE REVIEW

File: companies/landespolizei-des-fuerstentums-liechtenstein.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "relevant-countries": [
+        "li"
+    ],
+    "categories": [
+        "public body"
+    ],
+    "name": "Landespolizei des Fürstentums Liechtenstein",
+    "address": "Gewerbeweg 4\nPostfach 684\n9490 Vaduz\nLiechtenstein",
+    "phone": "+423 236 71 11",
+    "email": "info@landespolizei.li",
```
Comment (baltpeter) [inline companies/landespolizei-des-fuerstentums-liechtenstein.json:12]: Considering the Regierungskanzlei is mentioned as the data protection officer in the privacy policy, I think we should use the `Datenschutz@regierung.li` email.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record. I think specifying `Datenschutz@regierung.li` as the email is better. What do you think?

**Comment by mal-tee** [comment]
To chime in: 

This address is related to several other institutions, see https://www.llv.li/inhalt/918657/amtsstellen/fachstelle-datenschutz

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**